### PR TITLE
Set all CORS tests request methods to OPTIONS

### PR DESCRIPTION
**Pull request by vitchor**

### CODE REVIEW

**Comment by msfeldstein** [comment]
Let's add a message to this explaining that the endpoint needs to respond to the OPTIONS method with the correct flags to make sure browsers can use it, to get around preflight restrictions: https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request

**Comment by msfeldstein** [comment]
Might also be a good usage for a helper function `ensureCORS(url)`

**Comment by vitchor** [comment]
Yup, sounds good. I'll get those changes ready ASAP. Thanks!

**Comment by tomerweller** [comment]
According to the [CORS spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Preflighted_requests) both the preflight request and the actual request should include the `Access-Control-Allow-Origin:*` header. Should we check both?

**Comment by msfeldstein** [comment]
yes

**Comment by vitchor** [comment]
Done. Is this what you had in mind @msfeldstein ?

**Comment by vitchor** [comment]
If that's good, I'll use it in the other CORS tests.

**Review by msfeldstein — APPROVED** [review]
LGTM once we swap the other tests to use it

**Comment by vitchor** [comment]
Already swapped the other tests to use it 👍

File: cases/util/ensureCORS.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { fetch } from "./fetchShim";
+
+export const ensureCORS = async (URL, httpVerb) => {
```
Comment (msfeldstein) [inline cases/util/ensureCORS.js]: for clarity, we can make httpVerb default to GET (it seems like thats the only thing used anyway)

**Comment by msfeldstein** [comment]
lets make httpVerb default to GET then go ahead and land this

File: cases/util/ensureCORS.js (hunk 1 above)
Comment (msfeldstein) [reply cases/util/ensureCORS.js]: `async (URL, httpVerb = "GET") => {`

**Comment by vitchor** [comment]
Done @msfeldstein

**Pull request merged**